keyboards/mt/ncr80/readme.md

```diff
@@ -0,0 +1,16 @@
+# NCR-80
```

Comment:
Needs to follow the readme template.

Reply:
This is the top level readme, both readmes under soldered and hotswap follow the template, and my thought process is that, well, as written, 

>For more information, read the readme.md in soldered/ or hotswap/, depending on the version you are interested in.